test-resources/puppetlabs/ssl_utils/examples/ssl/crl_validation/other-3-cert-chain.pem

```diff
@@ -0,0 +1,57 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Was this needed to generate the `other-crl-chain`? Might be worth not keeping these around as a file, since they seem to be unused?

Reply:
Seeing this gave me the idea to double check that the `other-crl-chain` is actually valid when used with its own certs, and it is. That being said, thinking about it more, I do still feel like the old error ("No CRLs for issuer") for that one test made more sense... You never figured out what's going on there?

Reply:
yeah, this was generated along with the `other-crl-chain`. I figured it might be useful to have the certs that generated those CRLs, but I'm happy to remove this if we don't think we need it.

But yeah, I still don't really know what's going on with the changed error message.. I did figure out that it's actually the `3-cert/crl-chain` that changed somewhere though, not the `other-*` ones. I could keep going back farther to try and figure out the state I was in when those were originally generated, but I'm not even positive that was a state that ultimately got committed. I tried to regenerate whenever I made updates, but since I was originally developing on non-FIPS and this message change only shows itself in FIPS mode, I could've easily squashed some state that I thought didn't change the behavior.

Reply:
Okay.

Reply:
@Magisus did you want me to remove this file?

Reply:
Mmm... no, thinking about it more, I guess it's nice to have the certs associated with those CRLs, like you were saying. Disregard that comment 🙂